archlinuxcn/lib32-llvm-git/PKGBUILD

```diff
@@ -4,7 +4,7 @@
 pkgbase=lib32-llvm-git
 pkgname=('lib32-llvm-libs-git' 'lib32-clang-git' 'lib32-llvm-git')
 pkgdesc='Low Level Virtual Machine (git version)'
-pkgver=16.0.0_r432160.786b503f66b1a
+pkgver=15.0.0_r425063.0bd645d3
```

Comment:
This pkgver is older than the previous one, is this expected? Or you should bump the epoch.

Reply:
There’s no other change to this file? Then this is only going to cause conflicts….